product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx

```diff
@@ -87,3 +87,81 @@ application_name=psql
 2020-05-08 10:42:17 IST DETAIL:  Connection matched pg_hba.conf line 84:
 "local   all             all               md5"
 ```
+
+### Obfuscating the LDAP password
+
+You can use LDAP authentication to authenticate database users. For more information , see [LDAP authentication](https://www.postgresql.org/docs/15/auth-ldap.html).
+
+The `ldapbindpasswd` is stored in plaintext in the pg_hba.conf file. It is possible to store an encrypted `ldapbindpasswd` by writing an extension to decrypt. This extension should contain a decryption function with the C signature. You can add this extension to the `shared_preload_libraries` parameter in postgresql.conf file. You can restart your server to load the changes in this parameter.
+
+```ini
+shared_preload_libraries = '$libdir/ldap_password_func'
+```
+
+This example shows the code of extension `ldap_password_func.c` with the function `rot13_passphrase` to decrypt the password:
+
+```c
+/*-------------------------------------------------------------------------
+ *
+ * ldap_password_func.c
+ *
+ * Loadable PostgreSQL module to mutate the ldapbindpasswd. This
```

Comment:
If you are going to keep this comment, rewrite it for clarity using the same terminology used above.